mypy/checker.py

```diff
@@ -1357,15 +1371,18 @@ def check_override(self, override: FunctionLike, original: FunctionLike,
         """
         # Use boolean variable to clarify code.
         fail = False
+        dunder_note = False
```

Comment:
Can we rename this variable to `op_method_wider_note` or something else that makes it clear this is related to operator methods, not dunder methods in general? (E.g. we don't display this note for dunder methods like `__len__` or `__init__`.)

Reply:
Good point!